modules/stanford_people_spotlight_views/stanford_people_spotlight_views.module

```diff
@@ -5,3 +5,43 @@
  */
 
 include_once 'stanford_people_spotlight_views.features.inc';
+/**
+ * Implements template_preprocess_views_view()
+ *
+ * @link https://api.drupal.org/api/views/theme%21theme.inc/function/template_preprocess_views_view/7.x-3.x
+ */
+
+function stanford_people_spotlight_views_preprocess_views_view(&$vars) {
+  // Get the current view info
+  $view = $vars['view'];
+
+  // Add CSS/JS based on view name
+  if ($view->name == 'stanford_ppl_spot_3_v_card') {
+    //Add CSS/JS based on current view display
+    if ($view->current_display == 'page_1') {
+      drupal_page_is_cacheable(FALSE);
```

Comment:
Why is the page being forced to no cache?

Reply:
There was an issue with views_load_more and the caching due to the exposed filters